tests/wdio/vaah-webdriverio/Assert.js

```diff
@@ -1,12 +1,16 @@
 const env = require('./../../../wdio.env');
 
+const envObj = new env();
+
+const params = envObj.getParams();
+
 class Assert{
 
-    pause()
+    async pause()
```

Comment:
@we-abhijeet-g001 this can accept custom values like
```js
async pause(seconds = null){

if(seconds === null){
seconds = params.is_human_pause;
}
}
```
update the method, if you did not understand, discuss with me